### PR TITLE
fix build

### PR DESCRIPTION
**Pull request by zhongweiy**
ref: http://stackoverflow.com/questions/25423999/could-not-find-com-google-android-gmsplay-services5-0-77

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -20,7 +20,7 @@ android {
 }
 
 dependencies {
-    compile 'com.google.android.gms:play-services:5.0.77'
+    compile 'com.google.android.gms:play-services:5.0.86'
```
Comment (ThrowDice) [inline app/build.gradle]: For building the app module only, it seems that we can just remove this line. Could you remove this gms dependency and rebuild again locally?

Comment (zhongweiy) [inline app/build.gradle]: Sure, I can give a try later.
2014 年 10 月 13 日 下午 12:34于 "ThrowDice" notifications@github.com写道：

Comment (zhongweiy) [inline app/build.gradle]: @ThrowDice, I've tested gms is not a necessary dependency for current project, it can be eliminated.

**Comment by zhongweiy** [comment]
commit updated.

**Comment by ThrowDice** [comment]
LGTM.

**Pull request closed without merging**